src/app/public/modules/lookup/lookup.component.html

```diff
@@ -27,59 +27,109 @@
-            skyAutocomplete
-            wrap="off"
-            [attr.aria-label]="ariaLabel"
-            [attr.aria-labelledby]="ariaLabelledBy"
-            [attr.placeholder]="placeholderText"
-            [autocompleteAttribute]="autocompleteAttribute"
-            [disabled]="disabled"
-            (blur)="onAutocompleteBlur()"
-          ></textarea>
+            [displayWith]="descriptorProperty"
+            [focusable]="false"
+            [messageStream]="tokensController"
+            [trackWith]="idProperty"
+            [tokens]="tokens"
+            (keyup)="onTokensKeyUp($event)"
+            (tokensChange)="onTokensChange($event)"
+            (focusIndexOverRange)="onTokensFocusIndexOverRange()"
+          >
+            <textarea
+              class="sky-form-control sky-lookup-input"
+              rows="1"
+              skyAutocomplete
+              wrap="off"
+              [attr.aria-label]="ariaLabel"
+              [attr.aria-labelledby]="ariaLabelledBy"
+              [attr.placeholder]="placeholderText"
+              [autocompleteAttribute]="autocompleteAttribute"
+              [disabled]="disabled"
+              (blur)="onAutocompleteBlur()"
+              (keydown)="inputKeydown($event, $event.target.value)"
+              (keyup)="inputKeyup($event)"
+            ></textarea>
+          </sky-tokens>
+          <div *ngIf="selectMode === 'single'"
+            class="sky-lookup-single-control"
+          >
+            <textarea
+              class="sky-form-control sky-lookup-input"
+              rows="1"
+              skyAutocomplete
+              wrap="off"
+              [attr.aria-label]="ariaLabel"
+              [attr.aria-labelledby]="ariaLabelledBy"
+              [attr.placeholder]="placeholderText"
+              [autocompleteAttribute]="autocompleteAttribute"
+              [disabled]="disabled"
+              (blur)="onAutocompleteBlur()"
+            ></textarea>
+          </div>
         </div>
+        <ng-container *ngIf="!inputBoxHostSvc && enableShowMore">
+          <ng-container *ngTemplateOutlet="buttonsTemplateRef"></ng-container>
+        </ng-container>
+        <ng-container *ngIf="!inputBoxHostSvc && !enableShowMore">
+          <ng-container *ngTemplateOutlet="insetButtonsTemplateRef"></ng-container>
+        </ng-container>
       </div>
     </sky-autocomplete>
   </div>
 </ng-template>
+
+<ng-template
+  #buttonsTemplateRef
+>
+  <div
+    class="sky-input-group-btn"
+  >
+    <button
+      class="sky-btn sky-btn-default sky-lookup-search-btn"
+      type="button"
+      [attr.aria-label]="'skyux_lookup_search_button_show_more' | skyLibResources"
+      (click)="onShowMoreClick()"
+    >
+      <sky-icon *skyThemeIf="'default'"
+        icon="search"
+      ></sky-icon>
+      <sky-icon *skyThemeIf="'modern'"
+        iconType="skyux"
+        icon="search"
+      ></sky-icon>
+    </button>
+  </div>
+</ng-template>
+
+<ng-template
+  #insetButtonsTemplateRef
```

Comment:
I know that you are following what input box calls these. However, should we give these clearer template tag names since in this case the one template doesn't include a button. These wouldn't be API breaking changes if we needed to tweak them in the future so maybe `showMoreButtonTemplateRef` and `searchIconTemplateRef`?

Reply:
Yep, I'm ok with that. Changed.